src/react/components/styleform/PointTab.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { SvgRadioButton } from 'oskari-ui';
+import { Form, Row } from 'antd';
+
+export const PointTab = (props) => {
+    return (
+        <>
+            <Row>
+                <Form.Item
+                    { ...props.formLayout }
+                    name='image'
+                    label='Ikoni'
```

Comment:
It's work in progress and all but labels should come from the localization/Message component.

Reply:
Added localisations to all components with `Message`.